woocommerce/woocommerce-gutenberg-products-block
# Add dynamic rendering + SSR to Best Selling Products block

See #586 – This PR updates the Best Selling Products block.

#### Accessibility

I finally tested with VoiceOver/keyboard and noticed we can still click the links here, which seems unexpected. I added an issue to Gutenberg about it: https://github.com/WordPress/gutenberg/issues/15799

### Screenshots

<img width="577" alt="Screen Shot 2019-05-23 at 4 33 56 PM" src="https://user-images.githubusercontent.com/541093/58284437-98865a00-7d78-11e9-93b3-ccb2b9b7038a.png">

### How to test the changes in this Pull Request:

1. Add a Best Selling Products block to a post before activating this PR
2. Build the PR
3. View your existing post, the block should still render correctly
4. Add a new Best Selling Products block to a post, it should work correctly – show the right products, allow for setting categories in the sidebar, toggling the content, etc.
5. View your post on the frontend, it should display same as the editor preview
